core/src/main/java/org/hibernate/ogm/type/impl/TypeTranslatorImpl.java

```diff
@@ -121,4 +135,40 @@ else if ( type instanceof org.hibernate.type.CollectionType ) {
 		}
 		throw log.unableToFindGridType( type.getClass().getName() );
 	}
+
+	/*
+	   We directly pass the converter "database" type as value to the Tuple
```

Comment:
Ah, right we could indeed reuse ORM logic to do Java type -> ORM Type and then do our regular Type -> GridType.
I don't even think we need anything specific, we could do it today.

(I'm glad I named it GridType and not Type, that would be massively confusing ;) )